### PR TITLE
#372 releases.xml: XSD schema, test files, XSL page transform

### PR DESCRIPTION
**Pull request by carlosmiranda**
#372: Added XSD schema, test files, and XSL page transform for `releases.xml`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#373](https://github.com/zerocracy/datum/issues/373) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#373](https://github.com/zerocracy/datum/issues/373) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1921254042); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: xsd/pmo/types.xsd [hunk 1]
```diff
@@ -46,4 +46,9 @@ SOFTWARE.
       <xs:minInclusive value="0"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:simpleType name="repo">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[A-Z0-9a-z.\-/]{3,}"/>
```
Comment (krzyk) [inline xsd/pmo/types.xsd]: @carlosmiranda github repo usually is a combination of github username + / + repo name.
Regexp for github username is `/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i` (javascript syntax, from https://github.com/shinnn/github-username-regex#githubusernameregex)

I'm not sure what is the repo name, we could assume it is similar to username for now.

**Comment by krzyk** [comment]
@carlosmiranda and xcop complains about xml formatting

**Comment by carlosmiranda** [comment]
@krzyk updated, please check. I couldn't quite make the full regex to work with XSD; I put in a more simplistic regex that misses some corner cases (namely the rules on starting/ending/consecutive hyphens).

**Comment by krzyk** [comment]
@carlosmiranda ok, should be good for now

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#373](https://github.com/zerocracy/datum/issues/373) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**